src/audio/AudioManager.h

```diff
@@ -50,21 +69,21 @@ class AudioManager {
 
     static AudioManager& instance();
 
-    void playMusic(const char * fileName);
-    void playEffect(const char * fileName);
+    void playMusic(const char *fileName);
+    void playEffect(const char *fileName);
 
 private:
     static AudioManager sInstance;
 
-    musicMap  _music;
-    chunkMap  _chunks;
+    musicMap  mapMusic;
+    chunkMap  mapChunk;
```

Comment:
[hungarian notation is forbidden](https://github.com/bcit-datacomm/4981_Linux_Game/wiki/Style-Guide#variable-naming) you can just call it `music` and `chunks`